pkg/client/client_venconn_test.go

```diff
@@ -200,48 +194,7 @@ func run(test testcase) func(t *testing.T) {
 		test.given = strings.ReplaceAll(test.given, "FAKE_TPP_URL", fakeTPP.URL)
 
 		var given []ctrlruntime.Object
-		given = append(given, testutil.Parse(rbac)...)
-		given = append(given, testutil.Parse(testutil.Undent(`
-			apiVersion: v1
-			kind: Secret
-			metadata:
-			  name: accesstoken
-			  namespace: venafi
-			stringData:
-			  accesstoken: VALID_ACCESS_TOKEN
-			---
-			apiVersion: v1
-			kind: Secret
-			metadata:
-			  name: apikey
-			  namespace: venafi
-			stringData:
-			  apikey: VALID_API_KEY
-			---
-			apiVersion: rbac.authorization.k8s.io/v1
-			kind: Role
-			metadata:
-			  name: venafi-connection-secret-reader
-			  namespace: venafi
-			rules:
-			- apiGroups: [""]
-			  resources: ["secrets"]
-			  verbs: ["get"]
-			  resourceNames: ["accesstoken", "apikey"]
-			---
-			apiVersion: rbac.authorization.k8s.io/v1
-			kind: RoleBinding
-			metadata:
-			  name: venafi-connection-secret-reader
-			  namespace: venafi
-			roleRef:
-			  apiGroup: rbac.authorization.k8s.io
-			  kind: Role
-			  name: venafi-connection-secret-reader
-			subjects:
-			- kind: ServiceAccount
-			  name: venafi-connection
-			  namespace: venafi`))...)
+		given = append(given, testutil.Parse(testutil.VenConnRBAC)...)
```

Comment:
Could you add a TODO to the VenConnRBAC definition? I think once we get the Makefile modules setup, we should generate this based on the Helm chart.

Reply:
Done. I've opened a ticket to track this TODO: https://venafi.atlassian.net/browse/VC-36331